src/api/Snapshot.ts

```diff
@@ -310,9 +311,16 @@ export class Snapshot extends API {
 
     for (const score of scores) {
       for (const address of Object.keys(score)) {
-        result[address.toLowerCase()] = (result[address.toLowerCase()] || 0) + Math.floor(score[address] || 0)
+        result[address] = (result[address] || 0) + Math.floor(score[address] || 0)
       }
     }
+
+    for (const addr of addresses) {
+      if (!result[addr]) {
+        result[addr] = 0
```

Comment:
If the idea of this is to initialize results for those addresses that aren't in score, I believe you can just move this up

```typescript
for (const addr of addresses) {
  result[addr] = 0
}

for (const score of scores) {
...
```